txscript/data/script_tests.json

```diff
@@ -1026,7 +1026,81 @@
 ["42", "SIZE 1 EQUALVERIFY 42 EQUAL", "NONE", "OK", "SIZE does not consume argument"],
 ["NOP", "SIZE 1", "NONE", "ERR_INVALID_STACK_OPERATION"],
 
-["Right bit rotation related test coverage"],
+["Bitwise logic opcode tests"],
+
+["INVERT test coverage"],
+["1", "INVERT -2 EQUAL", "NONE", "OK"],
```

Comment:
This section needs a lot more tests.  Specifically surrounding numeric semantics.  Take a look at the `ROTL` coverage for a guide.

Also, need comments for all added tests.